tests/pkg_aversiveplusplus/Makefile

```diff
@@ -1,6 +1,11 @@
 APPLICATION = pkg_aversiveplusplus
 include ../Makefile.tests_common
 
+# temporarily disable for CI
+ifeq ($(RIOT_CI_BUILD), 1)
```

Comment:
when I execute `make RIOT_CI_BUILD=1` with this PR then it still builds (clones recursively) for me and outputs before the build starts:
```
The selected BOARD=native is not whitelisted: none


EXPECT ERRORS!
```